docs/_layouts/home.html

```diff
@@ -12,6 +12,9 @@
     .querySelectorAll(".markdown-body a[title]:not([href^=http])")
     .forEach((a) => {
       a.innerText = a.title;
+      if (a.href.endsWith('.md') || a.href.endsWith('.markdown')) {
```

Comment:
Markdown file extensions would probably be a good candidate to include into user config? Default values could maybe resemble this https://superuser.com/a/285878 or some subset of them?

Reply:
If we are going towards allow-listing markdown extensions, `.mdx` should definitely be there.

I know some of the other extensions in the ecosystem don't support this yet, but we want to be future compatible.

Reply:
ok, pushed a couple of changes to address those concerns. TIL about Jekyll templates :)